### ISSUE TITLE
android example solutions for selfie-segmentation

### ISSUE THREAD

**Issue by GreenVegetables** [google/mediapipe#3469]
Could you please provide an example solution for selfie-segmentation. Currently only facedetection, facemesh, hands. I really need selfie-segmentation example solution, please

**Comment by kuaashish** [comment]
Hi @GreenVegetables, We have Android examples implementation for selfie-segmentation as mentioned [here](https://google.github.io/mediapipe/). 
You can build and run Android examples by targeting the following path in the Android Studio:

`mediapipe/examples/android/src/java/com/google/mediapipe/apps/selfiesegmentationgpu/BUILD`

Please refer this issue [#3453  (comment)](https://github.com/google/mediapipe/issues/3453#issuecomment-1162845803) for more information.

Thank you!

**Comment by GreenVegetables** [comment]
Hello @kuaashish 

I have used the AAR of selfie-segmentation in my project and it is working fine. All contents of the package com.google.mediapipe are included in the AAR. like this:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/9369149/176104214-50742ebd-f4c6-4764-b58e-84b7f4853c24.png">


At the same time, I want to use facemesh with solution api way. In build.gradle, I added
```kotlin
implementation("com.google.mediapipe:solution-core:0.8.10.1-alpha-1")
implementation("com.google.mediapipe:facemesh:0.8.10.1-alpha-1")
```
When I run the project, it reports an error like this:

<img width="1120" alt="image" src="https://user-images.githubusercontent.com/9369149/176104477-407affad-f68c-4ce3-97e1-f602ca115565.png">

Because the solution-core dependencies also contain these：

<img width="520" alt="image" src="https://user-images.githubusercontent.com/9369149/176104728-3041cc7f-95fe-448d-826e-41476c08b86b.png">


How can I use selfie-segmentation and facemesh at the same time. please help me. thanks

**Comment by kuaashish** [comment]
Hi @GreenVegetables, Could you just open another issue with the second question to use two MediaPipe solutions at the same time. Thank you!

**Comment by GreenVegetables** [comment]
Hi @kuaashish 
I opened a new issue, it's [here](https://github.com/google/mediapipe/issues/3472)

**Comment by kuaashish** [comment]
Hi @GreenVegetables, Thanks for raising a separate issue for another query. Can we move this issue to close status as resolved and following #3472.
Thank you!

**Comment by GreenVegetables** [comment]
Hi @kuaashish , Thanks for following up and responding to my question. I will close this issue.
